### ISSUE TITLE
Info/suggestions

### ISSUE THREAD

**Issue by deoliveira86** [grp-bork/gunc#2]
Hello,
Thanks for developing the tool. I have some questions and/or suggestions:

1. In the gunc usage documentation: https://grp-bork.embl-community.io/gunc/usage.html it seems that the help --input_dir and --input_file contain the wrong description of their functionality (it is swapped actually).
2. There is a typo in containing (contining)
3. Could you please add an example of how to run gunc using already a protein file (--gene_calls input). I have been trying to run the sofware and an error message keeps appearing (and I don't know how to solve it). I have tried all different combinations of parameters but the execution with gunc using the .faa file is always crashing.
4. Is there a way to retrieve the scaffolds' name which contain the "contaminated" annotations?

```
[START] 11:41:09 2020-12-18
Traceback (most recent call last):
  File "/home/aloliveira/anaconda3/envs/gunc/bin/gunc", line 10, in <module>
    sys.exit(main())
  File "/home/aloliveira/anaconda3/envs/gunc/lib/python3.9/site-packages/gunc/gunc.py", line 512, in main
    run(args)
  File "/home/aloliveira/anaconda3/envs/gunc/lib/python3.9/site-packages/gunc/gunc.py", line 403, in run
    check_for_duplicate_filenames(fnas, args.file_suffix)
UnboundLocalError: local variable 'fnas' referenced before assignment
```

**The command line that I've used: gunc run -g Gene_predictions.faa -r gunc_db_2.0.4.dmnd**

Swapped fields concerning the --input_dir and --input_file parameters

```
GUNC RUN
Run chimerism detection.

Required Flags
--db_file Path to the GUNC database file. Can be set as environment variable GUNC_DB.

One of the following is required. If contigs (fna) are supplied the gene calls will be done using prodigal with option “-p meta”.

**--input_dir Input file in FASTA fna format.**

--file_suffix Only needed if suffix of files in --input_dir is not the default .fa.

**--input_file Input file contining paths to FASTA fna format files.**

--input_fna Input file in FASTA fna format.

--gene_calls Input genecalls FASTA faa format.

--use_species_level Allow species level to be picked as maxCSS. Default: False

--min_mapped_genes Dont calculate GUNC score if number of mapped genes is below this value. Default: 11
```

Thanks again for this really useful tool,
André

**Comment by fullama** [comment]
Thanks for the reports! 

1 - 100% right - fixed
2 - also fixed
3 - Thats my fault, you were running it correctly - thats a bug I introduced last minute by fixing another bug, I have fixed it and will push a new release later today - sorry about that
4 - not yet unfortunately but keep an eye out for updates

**Comment by fullama** [comment]
i have released `gunc v1.0.1` which should fix the problem with running from gene calls.. Thanks for letting me know about this one!
Try it out and reopen this issue if it doesn't work for you..

I plan to add better support for running from gene calls in the future.. its a bit limited right now.

Any other feedback is always welcome.!
